WordPress/Classes/ViewRelated/Post/PostNoticeViewModel.swift

```diff
@@ -271,6 +278,22 @@ struct PostNoticeViewModel {
                                                           comment: "Title of notification displayed when a post has failed to upload.")
         static let changesWillBeUploaded = NSLocalizedString("Changes will be uploaded next time your device is online",
                                                              comment: "Text displayed in notice after the app fails to upload a post.")
+        static let willAttemptToPublishLater = NSLocalizedString("Post couldn't be published. We'll try again later",
```

Comment:
2 questions on these. 
1. If these are the sam strings as in PostCardStatusViewModel, can we have them in one places so we'll only have to maintain once. 
2. I'm not loving the "Post couldn't be drafted" / "Post not drafted" and "Couldn't preform operation"... seems weird to me. Can we reevaluate the copy here, @osullivanchris ?

Reply:
1. Moved!
2. I updated the strings, it was wrong - my mistake. @osullivanchris if you want to review the strings you can check them here: https://github.com/wordpress-mobile/WordPress-iOS/blob/6eb538798f260ec00128f1221c54ef8bb1ce0648/WordPress/Classes/ViewRelated/Post/AutoUploadMessages.swift